database-commons/src/main/java/io/cdap/plugin/util/DBUtils.java

```diff
@@ -258,8 +257,7 @@ public static Object transformValue(int sqlType, int precision, int scale,
           return ((Number) original).intValue();
         case Types.NUMERIC:
         case Types.DECIMAL:
-          BigDecimal decimal = (BigDecimal) original;
-          return new BigDecimal(decimal.unscaledValue(), scale, new MathContext(precision));
```

Comment:
left a question in the other PR about why this is no longer needed

Reply:
These are SQL scale/precision and original instance will have the same scale/precision values in most cases(except of Oracle). We [handle this](https://github.com/data-integrations/database-plugins/pull/46/files#diff-08960aa54a73eedac06b8583a0c5e81bR133) by passing `scale` parameter:
```
BigDecimal decimal = resultSet.getBigDecimal(columnIndex, scale);
```